### PR TITLE
Add support for AirQualitySensor

### LINKED ISSUE #2574: Add support for AirQualitySensor
This includes ConcentrationProperty and DensityProperty

### PR DESCRIPTION
**Pull request by tim-hellhake**
resolves #2574

### CODE REVIEW

File: static/js/components/capability/air-quality-sensor.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+ * AirQualityCapability
+ *
+ * A bubble showing a air quality sensor icon.
```
Comment (mrstegeman) [inline static/js/components/capability/air-quality-sensor.js:4]: nit: "an air quality"

File: static/js/schema-impl/capability/air-quality-sensor.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
+/**
+ * AirQuality sensor.
+ *
+ * UI element representing a airquality sensor.
```
Comment (mrstegeman) [inline static/js/schema-impl/capability/air-quality-sensor.js:4]: nit: "an air quality"

File: static/js/schema-impl/capability/air-quality-sensor.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+   */
+  findProperties() {
+    this.airqualityProperty = null;
+
+    // Look for properties by type first.
+    for (const name in this.displayedProperties) {
+      const type = this.displayedProperties[name].property['@type'];
+
+      if (type === 'ConcentrationProperty') {
+        this.airqualityProperty = name;
```
Comment (mrstegeman) [inline static/js/schema-impl/capability/air-quality-sensor.js:46]: nit: `airQualityProperty`

**Review by mrstegeman — CHANGES_REQUESTED** [review]
I think my biggest issue is that we just pick the first concentration/density property to display, but that's what we do for other things as well. Just wish there was a better approach.

**Review by mrstegeman — APPROVED** [review]

**Comment by tim-hellhake** [comment]
> I think my biggest issue is that we just pick the first concentration/density property to display, but that's what we do for other things as well. Just wish there was a better approach.

Yeah, me too.
But as long as one device is tied to one _buble_, there is not much we can do.
Maybe we need something like the rule editor where you first pick the device and then the property you want to display.

**Comment by mrstegeman** [comment]
You mean like in #1826?

**Pull request merged**